src/client/app/components/UIOptionsComponent.jsx

```diff
@@ -4,11 +4,42 @@
 
 import React from 'react';
 import Slider from 'react-rangeslider';
+import { MultiSelect } from 'react-selectize';
+import 'react-selectize/themes/bootstrap3.css';
 import moment from 'moment';
 import 'react-rangeslider/lib/index.css';
 import { chartTypes } from '../reducers/graph';
 import ExportContainer from '../containers/ExportContainer';
 
+// Signifies that the containing object represents a meter
+const DATA_TYPE_METER = 'DATA_TYPE_METER';
```

Comment:
You can define these in the constructor, and bind it to `this`.

Reply:
These constants aren't associated with the component at all; I have them in that file because we currently don't have a better place to put generic constants. Making them associated constants would be very un-OOP.

Reply:
Is `utils/` a more appropriate place for these unassociated constants?